--- emacs-21.1/src/s/gnu-linux.h.ppc	Fri Nov 23 18:15:03 2001
+++ emacs-21.1/src/s/gnu-linux.h	Fri Nov 23 18:16:04 2001
@@ -190,7 +190,7 @@
    not -z combreloc is the default.  GNU ld ignores unknown -z KEYWORD
    switches, so this also works with older versions that don't implement
    -z combreloc.  */
-#define LD_SWITCH_SYSTEM_TEMACS -z nocombreloc
+#define LD_SWITCH_SYSTEM_TEMACS
 #endif /* __ELF__ */
 
 /* As of version 1.1.51, Linux did not actually implement SIGIO.
