--- emacs-22.3/src/xfns.c.toolbar	2008-01-08 05:29:40.000000000 +0100
+++ emacs-22.3/src/xfns.c	2009-04-13 11:14:32.000000000 +0200
@@ -3315,7 +3315,8 @@
 
   x_default_parameter (f, parms, Qmenu_bar_lines, make_number (1),
 		       "menuBar", "MenuBar", RES_TYPE_NUMBER);
-  x_default_parameter (f, parms, Qtool_bar_lines, make_number (1),
+  /* (pixel) Hide toolbar since it's not useful */
+  x_default_parameter (f, parms, Qtool_bar_lines, make_number (0),
 		       "toolBar", "ToolBar", RES_TYPE_NUMBER);
   x_default_parameter (f, parms, Qbuffer_predicate, Qnil,
 		       "bufferPredicate", "BufferPredicate",
